ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -486,12 +505,13 @@ class FamilyReads extends React.PureComponent {
 
 }
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state, ownProps) => ({
   igvSamplesByFamilySampleIndividual: getIGVSamplesByFamilySampleIndividual(state),
   sortedIndividualByFamily: getSortedIndividualsByFamily(state),
   familiesByGuid: getFamiliesByGuid(state),
   projectsByGuid: getProjectsByGuid(state),
   genesById: getGenesById(state),
+  spliceOutliersByFamily: getSpliceOutliersByChromFamily(state)[ownProps.variant?.chrom],
```

Comment:
would save computation if you only call `getSpliceOutliersByChromFamily` if there is a variant prop:
```
spliceOutliersByFamily: ownProps.variant && getSpliceOutliersByChromFamily(state)[ownProps.variant.chrom]
```